changelog/add-6034-feature-flag-bnlp

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
@KarlisJ I'd mark this as `patch` and maybe use the `dev` type. This comment is in the context of this [conversation](p1683298938613009-slack-CGGCLBN58) in Slack.